examples/govpp_call/main.go

```diff
@@ -48,103 +51,100 @@ import (
 // HTTP and Log), GOVPP, resync plugin and example plugin which demonstrates GOVPP call functionality.
 func main() {
 	// Init close channel to stop the example
-	closeChannel := make(chan struct{}, 1)
+	exampleFinished := make(chan struct{}, 1)
 
-	f := vpp.Flavor{}
+	// Start Agent with ExampleFlavor (combinatioplugin.GoVppmux, n of ExamplePlugin & reused cn-infra plugins)
+	flavor := ExampleFlavor{GovppExample: ExamplePlugin{closeChannel: &exampleFinished}}
+	agent := core.NewAgent(log.DefaultLogger(), 15*time.Second, append(flavor.Plugins())...)
+	core.EventLoopWithInterrupt(agent, exampleFinished)
+}
 
-	// Example plugin (GOVPP call)
-	examplePlugin := ExamplePlugin{}
-	examplePlugin.GoVppmux = &f.GoVPP
+/**********
+ * Flavor *
+ **********/
 
-	namedExamplePlugin := &core.NamedPlugin{PluginName: PluginID, Plugin: &examplePlugin}
+// ETCD flag to load config
+func init() {
```

Comment:
you can remove this init(), the flag have should be registered already

please test that after the change in runtime